.travis-build.sh

```diff
@@ -7,6 +7,15 @@ bootstrap_header() {
   echo -e "${ANSI_YELLOW}$1${ANSI_RESET}"
 }
 
+travis_fold() {
```

Comment:
See also here: https://docs.travis-ci.com/user/customizing-the-build/#Implementing-Complex-Build-Steps -> calling the script with `set -ev`